README.rst

```diff
@@ -67,8 +67,7 @@ templates, add this to your app's ``urls.py``:
         Server,
         fields='__all__',
         menus=['main'],
-        allow=lambda view,
-        user: return True, # Default requires is_staff !
+        allow=lambda view, user: return True, # Default requires is_staff!
```

Comment:
Yes, just the lambda bit. Functions is fine. But now the return:
`allow=lambda view, user: True`

https://docs.python.org/3/reference/expressions.html#lambda